jekyll/_includes/snippets/runner/ja/launch-agent-download.adoc

```diff
@@ -89,13 +89,19 @@ endif::[]
 | 1.0.29477-605777e
 
 | 3.4
-| 1.0.48283-583799b
+| 1.1.73695-40bf772
 
 | 4.0
-| 1.0.48283-583799b
+| 1.1.73695-40bf772
 
 | 4.1
-| 1.0.48283-583799b
+| 1.1.73695-40bf772
+
+| 4.2
+| 1.1.73695-40bf772
+
+| 4.3
+| 1.1.73189-8792751
```

Comment:
@ryanwohara, I meant to hide your comment as resolved instead of delete 🤦...

But the 4.3 should now be [updated to a current compatible version](https://circleci.slack.com/archives/C03F8RBUMM1/p1702558123548619) with the Go 1.21.5 patch. Thanks for catching that